psse/psse-model/src/main/java/com/powsybl/psse/model/PsseRawReader.java

```diff
@@ -73,6 +291,24 @@ public void handleError(DataProcessingException error, Object[] inputRow, Parsin
         return beans;
     }
 
+    private static String detectDelimiter(String record) {
+        CsvParserSettings settings = new CsvParserSettings();
+        settings.setHeaderExtractionEnabled(false);
+        settings.setQuoteDetectionEnabled(true);
+        settings.setDelimiterDetectionEnabled(true, ',', ' '); // sequence order is relevant
+        settings.setProcessorErrorHandler(new RetryableErrorHandler<ParsingContext>() {
+            @Override
+            public void handleError(DataProcessingException error, Object[] inputRow, ParsingContext context) {
+                LOGGER.error(error.getMessage());
+            }
+        });
```

Comment:
Should we extract these lines into a method to avoid duplication (see parseRecordsHeader above)

Reply:
Done.